### ISSUE TITLE
Can't use the same textures in spine animation (DEF-2709)

### ISSUE THREAD

**Issue by AGulev** [defold/editor2-issues#1866]
In spine:
![2018-05-04_17-18-59](https://user-images.githubusercontent.com/2209596/39633042-82e95266-4fbf-11e8-98bc-ed7d5e9a52c0.png)

In defold:
![2018-05-04_17-18-14](https://user-images.githubusercontent.com/2209596/39632922-2613e8c6-4fbf-11e8-81f2-dfc72fda7d7b.jpg)

Sources:
[spine_anim.zip](https://github.com/defold/editor2-issues/files/1974750/spine_anim.zip)
<hr/>

<table>
<tr><td>Build time</td><td>2018-05-03T10:29:15.219020</td></tr>
<tr><td>Defold channel</td><td>editor-alpha</td></tr>
<tr><td>Defold editor sha</td><td>e7472d566b3a0f0d492ab52ce6cbb279a47602be</td></tr>
<tr><td>Defold engine sha</td><td>32ba5eaa128e4b54113fb36eb4b9eff33d8582b3</td></tr>
<tr><td>Defold version</td><td>1.2.126</td></tr>
<tr><td>GPU</td><td>NVIDIA GeForce GT 750M OpenGL Engine</td></tr>
<tr><td>GPU Driver</td><td>2.1 NVIDIA-10.17.5 355.10.05.45f01</td></tr>
<tr><td>Java version</td><td>1.8.0_102-b14</td></tr>
<tr><td>OS arch</td><td>x86_64</td></tr>
<tr><td>OS name</td><td>Mac OS X</td></tr>
<tr><td>OS version</td><td>10.12.6</td></tr>
<table>

**Comment by andsveking** [comment]
Hi!

Is this a project (and branch) that anyone of us has access to? I would like to try it in the latest beta (ed1+bob+engine). We did a major refactoring of the Spine runtime and build pipeline, would be interesting if it handles this in a better way...

**Comment by AGulev** [comment]
no, project do not avaliable for you, but zip file with a spine anim and spine sources attached

**Comment by Dragosha** [comment]
I don't know why, but Defold uses "region" as image name instead "image path". So for quick solved this issue just use the same region name for images. 
Example:
![2018-05-04_21-43-02](https://user-images.githubusercontent.com/842807/39640384-4470fec4-4fe4-11e8-904a-638bf3c6aacd.png)

**Comment by AGulev** [comment]
@Dragosha thank you! We'll do this.

But I think using of "region" (name) instead of "path" still an issue
